### DISCUSSION TITLE
PatchTool 6.15 6/28

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#185] [Announcements]
修正翻譯錯誤與部分相容性問題。

「挑戰沙斯塔夏之人」主線任務我目前透過昔日重現模式做沒有問題，如果有人仍有問題可再回報。

檔案：
- FFXIVChnTextPatch-0628.rar：本體。一般而言，下載這個即可。JRE縮小過。

選用：
- rawexd-zhcn-0628.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- [第三方4K字體包](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.53/font-4k.rar)：部分解析度設定下會出現字體錯位等情形，請自行照需求使用。解壓後改名為font放到resource資料夾下（取代原資料夾）。
- [FFXIV-jar-0211.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.56/FFXIV-jar-0211.rar)：如果你有遇到JNI error，請參考 #90 放到與exe同位置使用。
- backup-0628.rar：乾淨的0a0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v2.64'>PatchTool 6.15 6/28</a>.</em>

**Comment by lcm123456** [comment]
大大好 請問是不是還有一些招式的翻譯還沒改掉? 例如武士的技能說明 辛苦了

**Reply by GpointChen** [reply]
是的

**Comment by UmaruPoi** [comment]
謝謝大大 感謝你的無私付出

**Comment by linpaul1129** [comment]
感謝您的漢化包，但我還是有遇到「挑戰沙斯塔夏之人」主線任務無法解的狀況，不過我還是試玩免費帳號不知道有沒有關係

**Reply by GpointChen** [reply]
那可能需要我之後創新角色來窮舉問題出在哪了

**Comment by linyuchin** [comment]
今天更新之後
重新漢化時會出現程式錯誤的 dialog 
log 檔中停在這裡
![image](https://user-images.githubusercontent.com/37316862/177364617-87285b3a-c480-4f7f-8da4-2273617c4ac5.png)

**Reply by linyuchin** [reply]
更新一下現況
剛剛有把 backup.tar 的檔案載下來 並還原回去
這時候再漢化是有成功的
但是因為這次更新有對伺服器做調整 導致我找不到我創建角色的伺服器 Orz

**Reply by okami2223** [reply]
漢化過程沒有遇到問題，反而是在近遊戲後不定時會有DX11錯誤發生，必須得整個遊戲修復後才正常

**Reply by linyuchin** [reply]
最一開始的時候有先用 5/22 ver. 的漢化包漢化過一次
失敗後才發現 6/28 有更新  才改用 6/28
不確定是不是因為有先用 5/22 ver. 的  才導致後續漢化都失敗
後來把整個遊戲重新安裝
再重上漢化之後OK了
DX11問題我朋友也有遇到 也是重新安裝遊戲後恢復正常
感謝大大 ><!

**Reply by GpointChen** [reply]
沒有還原過就更新或再次進行漢化的話很容易出問題，有解決就好
